package.json

```diff
@@ -125,16 +125,16 @@
     "@types/react-portal": "^4.0.4",
     "@types/svg4everybody": "^2.1.2",
     "axe-core": "^4.4.3",
-    "chromatic": "^6.13.1",
+    "chromatic": "^6.14.0",
     "codecov": "^3.8.3",
     "copyfiles": "^2.4.1",
-    "eslint": "^8.29.0",
+    "eslint": "^8.31.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-jest": "^27.1.7",
+    "eslint-plugin-jest": "^27.2.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-storybook": "^0.6.4",
     "eslint-plugin-testing-library": "^5.9.1",
-    "flowgen": "^1.20.1",
+    "flowgen": "^1.21.0",
```

Comment:
🤔 We can probably delete flowgen. I probably forgot to remove this dep when removing the flow type generation script

Reply:
Got it, will remove from this pr before landing